package.json

```diff
@@ -4,7 +4,7 @@
   "description": "config and resource loader",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "tape test.js"
```

Comment:
@fedor 

Try replacing tape keyword with babel-tape-runner for using ES6 based testing.